Dmf/Framework/DmfCore.c

```diff
@@ -1440,10 +1442,16 @@ Return Value:
         //
         if (DmfModuleAttributes->DynamicModuleImmediate)
         {
+            // Dynamic Module Path:
             // Remember it is Dynamic Module so it can be automatically closed prior to destruction.
-            // (Client no longer has access to Open/Close API.)
             //
             dmfObject->DynamicModuleImmediate = TRUE;
+            // Give Client the resultant Module Handle:
+            // PostOpen callback may need to compare contents of the address of the Module it
+            // passed with the Module handle passed in the callback. So, set this now before 
+            // PostOpen callback happens.
+            //
+            *DmfModule = dmfModule;

```

Comment:
Will this always be non NULL?

Reply:
I added it back. I think it is unlikely, but no impossible a Client would pass NULL.